docs/tuning-apps/coll-tuned.rst

```diff
@@ -128,10 +144,20 @@ for details.
 One may provide rules for as many collectives, communicator sizes, and message
 sizes as desired. Simply repeat the sections as needed and adjust the relevant
 count parameters.  One must always provide a rule for message size of zero.
-Message size rules are expected in ascending order.  The last two parameters in
-the rule may or may not be used and have different meaning depending on the
-collective and algorithm.  As of writing not all of the relevant control
-parameters can be set by the rules file (See issue #12589).
+Message size rules are expected in ascending order. The last parameters in the
+message size rule may or may not be used and have different meaning depending
+on the collective and algorithm. The first two parameters in the rule following
+the algorithm id, `topo` and `segment size`, are always required. In version 2
+of the file format a third parameter, `max requests`, may also be provided. A
+release of OpenMPI at least v5.0.7 or v4.1.8 is required for version 2 features.
```

Comment:
I know this is a nit, but the official name of the project is "Open MPI" (with a space).  We have to get that correct in the docs.  😄 

```suggestion
release of Open MPI at least v5.0.7 or v4.1.8 is required for version 2 features.
```

Reply:
fixed